### PR TITLE
Add warning class to delete

### LINKED ISSUE #6847: Delete buttons not red in Automation/Script editors
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

image

## Expected behavior

These should be red like everywhere else

## Steps to reproduce

1. Go to Automation Editor or Script Editor
2. Click the menu on a trigger, condition, or action
3. Look at the Delete button

## Environment

- Home Assistant release with the issue: 0.115.0b0
- Last working Home Assistant release (if known): Not sure, I don't use these much, just hunting bugs 😛 
- Browser and browser version: Chrome stable
- Operating system: macOS

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

### PR DESCRIPTION
**Pull request by ludeeus**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #6847
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/automation/action/ha-automation-action-row.ts [hunk 1]
```diff
@@ -173,7 +173,7 @@ export default class HaAutomationActionRow extends LitElement {
                   "ui.panel.config.automation.editor.actions.duplicate"
                 )}
               </mwc-list-item>
-              <mwc-list-item>
+              <mwc-list-item class="error">
```
Comment (bramkragten) [inline src/panels/config/automation/action/ha-automation-action-row.ts:176]:
```suggestion
              <mwc-list-item class="warning">
```

File: src/panels/config/automation/condition/ha-automation-condition-row.ts [hunk 2]
```diff
@@ -86,7 +86,7 @@ export default class HaAutomationConditionRow extends LitElement {
                   "ui.panel.config.automation.editor.actions.duplicate"
                 )}
               </mwc-list-item>
-              <mwc-list-item>
+              <mwc-list-item class="error">
```
Comment (bramkragten) [inline src/panels/config/automation/condition/ha-automation-condition-row.ts:89]:
```suggestion
              <mwc-list-item class="warning">
```

File: src/panels/config/automation/trigger/ha-automation-trigger-row.ts [hunk 3]
```diff
@@ -118,7 +118,7 @@ export default class HaAutomationTriggerRow extends LitElement {
                   "ui.panel.config.automation.editor.actions.duplicate"
                 )}
               </mwc-list-item>
-              <mwc-list-item>
+              <mwc-list-item class="error">
```
Comment (bramkragten) [inline src/panels/config/automation/trigger/ha-automation-trigger-row.ts:121]:
```suggestion
              <mwc-list-item class="warning">
```

**Comment by bramkragten** [comment]
I don't see the styles for the error or warning class?

**Comment by ludeeus** [comment]
That's because they are not there ;)
They are defined in `haStyle`, both with `--error-color`

**Comment by bramkragten** [comment]
> That's because they are not there ;)
> They are defined in `haStyle`, both with `--error-color`

But I was missing haStyle too 😄

**Review by bramkragten — APPROVED** [review]

**Comment by ludeeus** [comment]
Yeah, was missing for the condition row :D

**Pull request merged**